src/gui/KM_InterfaceDefaults.pas

```diff
@@ -52,12 +52,12 @@   TKMUserInterfaceCommon = class
     property MyControls: TKMMasterControl read fMyControls;
     procedure ExportPages(aPath: string); virtual; abstract;
 
-    procedure KeyDown(Key: Word; Shift: TShiftState); virtual; abstract;
+    function KeyDown(Key: Word; Shift: TShiftState): Boolean; virtual; abstract;
```

Comment:
How about if we change this and alikes into "procedure KeyDown(Key: Word; Shift: TShiftState; var aHandled: Boolean); virtual; abstract;" ?